apps/openmw/mwmechanics/spellcasting.hpp

```diff
@@ -4,6 +4,7 @@
 #include <components/esm/loadskil.hpp>
 
 #include "../mwworld/ptr.hpp"
+#include "../mwworld/esmstore.hpp"
```

Comment:
Seems like we are including much more than necessary here. components/esm/effectlist.hpp would be enough, right?